### PR TITLE
fix(input): add aria-label/labelledby

### PR DESCRIPTION
**Pull request by nadiia**

### CODE REVIEW

File: src/input/__tests__/__snapshots__/input.test.js.snap [hunk 1]
```diff
@@ -3,6 +3,8 @@
 exports[`Input - basic functionality: input has correct props 1`] = `
 Object {
   "aria-invalid": false,
+  "aria-label": undefined,
```
Comment (gergelyke) [inline src/input/__tests__/__snapshots__/input.test.js.snap]: do we want to provide a value for the tests?

Comment (nadiia) [reply src/input/__tests__/__snapshots__/input.test.js.snap]: updated, should have default `null`

Comment (nadiia) [reply src/input/__tests__/__snapshots__/input.test.js.snap]: + updated e2e setting

**Comment by chasestarr** [comment]
fyi i added this same feature for `radio` here https://github.com/uber-web/baseui/blob/master/src/radio/types.js#L33-L42 camelCased may want to stay consistent?

**Comment by nadiia** [comment]
> fyi i added this same feature for `radio` here https://github.com/uber-web/baseui/blob/master/src/radio/types.js#L33-L42 camelCased may want to stay consistent?

The `aria-label` is the original attribute and prop name, what do you think about sticking to it to avoid a confusion?

**Comment by gergelyke** [comment]
can we support both in all the components? would that make sense?

Ideally, we'd stay consistent but at the same time we would not introduce a breaking change

**Comment by chasestarr** [comment]
i understand that it's the name react uses with elements, but it honestly feels like a strange deviation on react's end. 

i'm fine with enabling both options for radio, making a note for future deprecation, and going with `aria-*` across the repo

**Comment by gergelyke** [comment]
following up on this, @nadiia / @chasestarr would one of you be up for sending a PR to the radio component to make it work consistently (with the deprecation notice)?

**Comment by chasestarr** [comment]
fyi i will be adding the radio change in my radio overrides pr today

**Pull request merged**